core/scss/components/button/button.scss

```diff
@@ -26,7 +26,7 @@
       $button__transition-speed,
       ease
     );
-
```

Comment:
Unecessary changes here.  Please remove these.

Reply:
> 
> 
> Commit mesage doesn't match change.

I updated PR with above request change. You can have a look.